### DISCUSSION TITLE
Alt hotkeys stopped working while Kitty is focused

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [kovidgoyal/kitty#8088] [Q&A] [answered]
Suddenly all my alt hotkeys stopped working while my Kitty terminal is focused. 
I'm using `skhd` and `aerospace`. From both tools hot keys doesn't work when Kitty is focused.
Any ideas what could it be and how to fix the issue?

I'm using Kitty 0.37.0 and macOS Sequoia 15.1.1.

**Comment by kovidgoyal** [comment, marked as answer]
You turned on secure text input turn it off again.

**Reply by wojciech-kulik** [reply]
Hmm, I didn't enable it intentionally, but maybe I did it by accident. Where can I find it?

**Reply by wojciech-kulik** [reply]
Oh, I found it, it's Kitty's feature. I thought you are talking about some macOS feature. Thank you! You saved my day!
